pet-store-vanilla-js/api.js

```
@@ -0,0 +1,62 @@
+const api = (function api(){
+    function getPet(petId) {
+        return fetchFromApi(`/pet/${petId}`, 'GET');
+    }
```

Throw an error instead of returning the `response` object.

---

I've used the non-exception error handling but it doesn't work well without TypeScript.

---

If the api response status code is not in the range 200-299, an error is thrown with the returned status code.